File: Programming-languages-and-laboratory/Java/Exercises/30_compito_java_2019_06_17/Giocatore.java
public class Giocatore {
	protected int codiceSquadra;
	protected String cognome, nome;
	protected boolean titolare;
	protected int eta, numeroMaglia;
	protected String ruolo;
	
	public Giocatore(int codiceSquadra, String cognome, String nome, boolean titolare, int eta, int numeroMaglia, String ruolo) {
		this.codiceSquadra = codiceSquadra;
		this.cognome = cognome;
		this.nome = nome;
		this.titolare = titolare;
		this.eta = eta;
		this.numeroMaglia = numeroMaglia;
		this.ruolo = ruolo;
	}
	
	public String toString() {
		return nome + "\t" + cognome + "\t" + eta + "\t" + numeroMaglia + "\t" + ruolo + "\t";
	}
}

File: II - Linguaggi di programmazione e laboratorio/Python/Exercises/35_compito_java_20191120/Giocatore.py
class Giocatore:
    def __init__(self, codice, cognome, nome, eta, maglia, ruolo, titolare):
        self._codice = codice
        self._cognome = cognome
        self._nome = nome
        self._eta = eta
        self._maglia = maglia
        self._ruolo = ruolo
        self._titolare = titolare

    def get_codice(self):
        return self._codice

    def __str__(self):
        return self._nome + " " + self._cognome + "\t" + str(self._eta) + "\t" + str(self._maglia) + "\t" + self._ruolo + "\t" + str(self._titolare)
